src/components/HomepageVideo.js

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import { SearchInput, useTranslation } from '@newrelic/gatsby-theme-newrelic';
+import { navigate } from '@reach/router';
+import curlyAndDotsBackground from './curlyAndDots.png';
+import curlyAndDotsBackgroundDarkmode from './curlyAndDotsDM.png';
+import InlineSignup from './InlineSignup';
+import { useMainLayoutContext } from './MainLayoutContext';
+
+const HomepageVideo = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const { t } = useTranslation();
+  const [sidebar] = useMainLayoutContext();
+  const [mobileBreakpoint, setMobileBreakpoint] = useState('1050px');
+
+  useEffect(() => {
+    if (sidebar) {
+      setMobileBreakpoint('1300px');
+    } else {
+      setMobileBreakpoint('1050px');
+    }
+  }, [sidebar]);
+
+  const PageContainer = styled.div`
+    padding-top: 7rem;
+    padding-bottom: 1rem;
+    width: 100%;
+    display: grid;
+    grid-template-areas: 'welcome video' 'break signup';
+    grid-template-columns: 2fr 3fr;
+    grid-gap: 1rem;
+    background-image: url(${curlyAndDotsBackground});
+    background-size: 70%;
+    background-repeat: no-repeat;
+    background-position: top -3vh right 20%;
+    .dark-mode & {
+      background-image: url(${curlyAndDotsBackgroundDarkmode});
+    }
+    @media screen and (max-width: ${mobileBreakpoint}) {
+      background-image: none;
+      display: flex;
+      flex-direction: column;
+      padding-top: 2rem;
+      .dark-mode & {
+        background-image: none;
+      }
+    }
+  `;
+  const WelcomeContainer = styled.div`
+    grid-area: welcome;
+    padding-right: 3rem;
```

Comment:
can we get rid of this on mobile? it'll just make the search bar and video look even when they're stacked
<img width="824" alt="Screen Shot 2023-02-27 at 3 25 49 PM" src="https://user-images.githubusercontent.com/26727669/221700017-aaed8ff7-7596-4efc-b931-648882cfff62.png">
vs:
<img width="825" alt="Screen Shot 2023-02-27 at 3 25 58 PM" src="https://user-images.githubusercontent.com/26727669/221700061-381fc6bb-aa53-4846-b977-e2db9ecab581.png">

Reply:
tis gone!